apps/strapi-cms/src/components/components/cards-item.json

```diff
@@ -31,6 +31,15 @@
       "enum": [
         "AccessAlarm"
       ]
+    },
```

Comment:
Wouldn't it be better to use an array?

Reply:
It absolutely would.

However Strapi limits nested components (which we need, to create an array) to two levels.

There is a workaround, but it's not well looked upon by Strapi themselves due to "[severe performance problems](https://forum.strapi.io/t/multiple-nested-repeatable-components/665/10)".

That's simply updating this file manually without using the Admin Panel.

I'm honestly not against it (unless these performance issues are really THAT severe).